source/game/field/Object.cc

```diff
@@ -0,0 +1,14 @@
+#include "Object.hh"
+
+#include <egg/math/Math.hh>
+
+namespace Field {
+
+Object::Object(const System::MapdataGeoObj &params) {
+    m_id = params.id();
+    m_pos = params.pos();
```

Comment:
To avoid the default c'tors, `m_pos`, `m_rot`, and `m_scale` should all be initialized via an initializer list.

Reply:
My initial problem with doing this is the fact that `m_rot` is not initialized directly to `params.rot()` - we apply `DEG2RAD` as well. I'll definitely consider it though.

Reply:
Syntactically that's perfectly fine to have in an initializer list, but I think I know what you mean